packages/wrangler/src/versions/deploy.ts

```diff
@@ -47,6 +48,10 @@ export const versionsDeployCommand = createCommand({
 		owner: "Workers: Authoring and Testing",
 		status: "stable",
 	},
+	behaviour: {
+		provideConfig: false,
```

Comment:
why don't we need `useConfigRedirect: true,` here too?

Reply:
Oh yes! We can.

Reply:
Done